assets/xml/archives/icon_item_static.xml

```diff
@@ -1,194 +1,222 @@
-        <Blob Name="icon_item_static_yar_Blob_093DC0" Size="0x0180" Offset="0x93DC0" />
-        <Blob Name="icon_item_static_yar_Blob_093F40" Size="0x0300" Offset="0x93F40" />
-        <Blob Name="icon_item_static_yar_Blob_094240" Size="0x06C0" Offset="0x94240" />
-        <Blob Name="icon_item_static_yar_Blob_094900" Size="0x06C0" Offset="0x94900" />
-        <Blob Name="icon_item_static_yar_Blob_094FC0" Size="0x0300" Offset="0x94FC0" />
-        <Blob Name="icon_item_static_yar_Blob_0952C0" Size="0x0300" Offset="0x952C0" />
-        <Blob Name="icon_item_static_yar_Blob_0955C0" Size="0x0500" Offset="0x955C0" />
-        <Blob Name="icon_item_static_yar_Blob_095AC0" Size="0x0400" Offset="0x95AC0" />
+        <!-- Item icons -->
+        <Texture Name="gItemIconOcarinaOfTimeTex" OutName="ocarina_of_time_icon" Format="rgba32" Width="32" Height="32" Offset="0x0000"/>
+        <Texture Name="gItemIconBowTex" OutName="bow_icon" Format="rgba32" Width="32" Height="32" Offset="0x1000"/>
+        <Texture Name="gItemIconFireArrowTex" OutName="fire_arrow_icon" Format="rgba32" Width="32" Height="32" Offset="0x2000" />
+        <Texture Name="gItemIconIceArrowTex" OutName="ice_arrow_icon" Format="rgba32" Width="32" Height="32" Offset="0x3000" />
+        <Texture Name="gItemIconLightArrowTex" OutName="light_arrow_icon" Format="rgba32" Width="32" Height="32" Offset="0x4000" />
+        <Texture Name="gItemIconFairyOcarinaTex" OutName="fairy_ocarina_icon" Format="rgba32" Width="32" Height="32" Offset="0x5000" />
+        <Texture Name="gItemIconBombTex" OutName="bomb_icon" Format="rgba32" Width="32" Height="32" Offset="0x6000" />
+        <Texture Name="gItemIconBombchuTex" OutName="bombchu_icon" Format="rgba32" Width="32" Height="32" Offset="0x7000" />
+        <Texture Name="gItemIconDekuStickTex" OutName="deku_stick_icon" Format="rgba32" Width="32" Height="32" Offset="0x8000" />
+        <Texture Name="gItemIconDekuNutTex" OutName="deku_nut_icon" Format="rgba32" Width="32" Height="32" Offset="0x9000" />
+        <Texture Name="gItemIconMagicBeansTex" OutName="magic_beans_icon" Format="rgba32" Width="32" Height="32" Offset="0xA000" />
+        <Texture Name="gItemIconSlingshotTex" OutName="slingshot_icon" Format="rgba32" Width="32" Height="32" Offset="0xB000" />
+        <Texture Name="gItemIconPowderKegTex" OutName="powder_keg_icon" Format="rgba32" Width="32" Height="32" Offset="0xC000" />
+        <Texture Name="gItemIconPictoBoxTex" OutName="picto_box_icon" Format="rgba32" Width="32" Height="32" Offset="0xD000" />
+        <Texture Name="gItemIconLensofTruthTex" OutName="lens_of_truth_icon" Format="rgba32" Width="32" Height="32" Offset="0xE000" />
+        <Texture Name="gItemIconHookshotTex" OutName="hookshot_icon" Format="rgba32" Width="32" Height="32" Offset="0xF000" />
+        <Texture Name="gItemIconGreatFairySwordTex" OutName="great_fairy_sword_icon" Format="rgba32" Width="32" Height="32" Offset="0x10000" />
```

Comment:
Hate to be That Guy:tm: again, but it's "Great Fairy's Sword":
![GLideN64_ZELDA_MAJORA'S_MASK_022](https://github.com/zeldaret/mm/assets/12245827/ea888887-eeea-4c8f-a414-c0ccfa9e5eef)

Reply:
The actual names of the items is the extraction I'm doing next, so I'll keep an eye out for any other inconsistencies/plurals